dsgrid/config/dimension_config.py

```diff
@@ -1,20 +1,8 @@
 import abc
 import logging
-import os
```

Comment:
LOL all these unused imports, must have been legacy from before refactoring?

Reply:
It is very difficult to remember to delete these after you remove their use. I blame Python.